### PR TITLE
Add open window endpoints

### PR DESCRIPTION
**Pull request by Blacksmoke16**
Add the documentation to the two endpoints that make use of the remoteClientUI scope.

### CODE REVIEW

File: docs/crest/character/char_marketDetails.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Open Market Details
+The characters openWindow marketdetails resource allows an application to open the market details window for a specific typeID in game.
+
+### Route
+``/characters/<characterID:characterIdType>/ui/openwindow/marketdetails/``
+
+### POST
+* Scope: `remoteClientUI`
+
+The typeID in the href of body does not matter and can remain 0.
```
Comment (CarbonAlabel) [inline docs/crest/character/char_marketDetails.md]: Who was it that originally came up with this? Sure, it works now, but placing the ID in the `href` as well is next to no extra work and a bit more future proof.

Comment (Blacksmoke16) [inline docs/crest/character/char_marketDetails.md]: https://forums.eveonline.com/default.aspx?g=posts&m=6535723#post6535723

"The ID in the URL actually doesn't matter... quark with the CREST framework."
- CCP FoxFour

Comment (CarbonAlabel) [inline docs/crest/character/char_marketDetails.md]: Don't know what else I was expecting ¯\\\_(ツ)\_/¯

Still think that it shouldn't be here. If nothing else, it sounds ridiculous, and as I said, almost no extra work.

Comment (Blacksmoke16) [inline docs/crest/character/char_marketDetails.md]: Meh, my as well.  Will add in a note that it doesn't matter but may be wise to include it for future proofing.

**Comment by polarina** [comment]
Thanks! Cherry-picked as aad593a8117654c392cfaab8abe903e480046cad.

**Pull request closed without merging**